downstream/assemblies/topologies/assembly-overview-tested-deployment-models.adoc

```diff
@@ -2,7 +2,11 @@
 
 = Overview of tested deployment models
 
-Red Hat tests {PlatformNameShort} {PlatformVers} with a defined set of topologies to give you opinionated deployment options. While it is possible to install the {PlatformNameShort} on different infrastructure topologies and with different environment configurations, Red Hat provides full support for the topologies outlined in this document.
+Red Hat tests {PlatformNameShort} {PlatformVers} with a defined set of topologies to give you opinionated deployment options. Red Hat recommends that customers always deploy all components of {PlatformNameShort} so that all features and capabilities are available for use without the need to take further action.
```

Comment:
Suggestion based on RH supplementary guide regarding using "Red Hat suggests": 

```suggestion
Red Hat tests {PlatformNameShort} {PlatformVers} with a defined set of topologies to give you opinionated deployment options. Based on results from these tests, always deploy all components of {PlatformNameShort} so that all features and capabilities are available for use without the need to take further action.
```